diff --git a/src/uct/cuda/base/cuda_iface.h b/src/uct/cuda/base/cuda_iface.h
index a1cb28a4e14..a7f9a3688f1 100644
--- a/src/uct/cuda/base/cuda_iface.h
+++ b/src/uct/cuda/base/cuda_iface.h
@@ -22,5 +22,6 @@ typedef enum uct_cuda_base_gen {
     UCT_CUDA_BASE_GEN_V100 = 7,
     UCT_CUDA_BASE_GEN_A100 = 8,
     UCT_CUDA_BASE_GEN_H100 = 9,
-    UCT_CUDA_BASE_GEN_B100 = 10
+    UCT_CUDA_BASE_GEN_B100 = 10,
+    UCT_CUDA_BASE_GEN_BRTX = 12  /* Blackwell RTX */
 } uct_cuda_base_gen_t;
diff --git a/src/uct/cuda/cuda_ipc/cuda_ipc_iface.c b/src/uct/cuda/cuda_ipc/cuda_ipc_iface.c
index ed09c8fa127..a7c169cf07a 100644
--- a/src/uct/cuda/cuda_ipc/cuda_ipc_iface.c
+++ b/src/uct/cuda/cuda_ipc/cuda_ipc_iface.c
@@ -197,5 +197,7 @@ static double uct_cuda_ipc_iface_get_bw()
     case UCT_CUDA_BASE_GEN_B100:
         return 800000.0 * UCS_MBYTE;
+    case UCT_CUDA_BASE_GEN_BRTX:
+        return 50000.0 * UCS_MBYTE;
     default:
         return 6911.0  * UCS_MBYTE;
     }
 }
