diff --git a/presto-native-execution/scripts/dockerfiles/centos-dependency.dockerfile b/presto-native-execution/scripts/dockerfiles/centos-dependency.dockerfile
index fc9c0c810f..bdc002d995 100644
--- a/presto-native-execution/scripts/dockerfiles/centos-dependency.dockerfile
+++ b/presto-native-execution/scripts/dockerfiles/centos-dependency.dockerfile
@@ -33,7 +33,12 @@ RUN bash -c "mkdir build && \
                  ../scripts/setup-adapters.sh && \
                  source ../velox/scripts/setup-centos9.sh && \
                  source ../velox/scripts/setup-centos-adapters.sh && \
-                 install_adapters && \
+                 # equivalent of install_adapters() but without problematic Hadoop SDK download
+                 install_adapters_deps_from_dnf && \
+                 install_s3 && \
+                 install_gcs-sdk-cpp && \
+                 install_azure-storage-sdk-cpp && \
+                 # continue as before
                  install_clang15 && \
                  install_cuda 12.8) && \
     rm -rf build"
