diff --git a/scripts/setup-centos-adapters.sh b/scripts/setup-centos-adapters.sh
index 6e8276c48..955502a41 100755
--- a/scripts/setup-centos-adapters.sh
+++ b/scripts/setup-centos-adapters.sh
@@ -79,8 +79,8 @@ function install_s3 {
 function install_adapters {
   run_and_time install_adapters_deps_from_dnf
   run_and_time install_s3
-  run_and_time install_gcs-sdk-cpp
-  run_and_time install_azure-storage-sdk-cpp
+  run_and_time install_gcs_sdk_cpp
+  run_and_time install_azure_storage_sdk_cpp
   run_and_time install_hdfs_deps
 }
 
diff --git a/scripts/setup-common.sh b/scripts/setup-common.sh
index e205f3980..a207a073c 100755
--- a/scripts/setup-common.sh
+++ b/scripts/setup-common.sh
@@ -298,7 +298,7 @@ function install_minio {
   ${SUDO} mv ./"${MINIO_BINARY_NAME}" "$INSTALL_PREFIX"/bin/
 }
 
-function install_gcs-sdk-cpp {
+function install_gcs_sdk_cpp {
   # Install gcs dependencies
   # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/packaging.md#required-libraries
 
@@ -347,7 +347,7 @@ function install_gcs-sdk-cpp {
     -DGOOGLE_CLOUD_CPP_ENABLE=storage
 }
 
-function install_azure-storage-sdk-cpp {
+function install_azure_storage_sdk_cpp {
   # Disable VCPKG to install additional static dependencies under the VCPKG installed path
   # instead of using system pre-installed dependencies.
   export AZURE_SDK_DISABLE_AUTO_VCPKG=ON
diff --git a/scripts/setup-macos.sh b/scripts/setup-macos.sh
index 5c05c0dcb..a10295a37 100755
--- a/scripts/setup-macos.sh
+++ b/scripts/setup-macos.sh
@@ -111,11 +111,11 @@ function install_s3 {
 }
 
 function install_gcs {
-  install_gcs-sdk-cpp
+  install_gcs_sdk_cpp
 }
 
 function install_abfs {
-  install_azure-storage-sdk-cpp
+  install_azure_storage_sdk_cpp
 }
 
 function install_hdfs {
diff --git a/scripts/setup-ubuntu.sh b/scripts/setup-ubuntu.sh
index 339b129bb..f50e78351 100755
--- a/scripts/setup-ubuntu.sh
+++ b/scripts/setup-ubuntu.sh
@@ -203,13 +203,13 @@ function install_s3 {
 function install_gcs {
   # Dependencies of GCS, probably a workaround until the docker image is rebuilt
   apt install -y --no-install-recommends libc-ares-dev libcurl4-openssl-dev
-  install_gcs-sdk-cpp
+  install_gcs_sdk_cpp
 }
 
 function install_abfs {
   # Dependencies of Azure Storage Blob cpp
   apt install -y openssl libxml2-dev
-  install_azure-storage-sdk-cpp
+  install_azure_storage_sdk_cpp
 }
 
 function install_hdfs {
